src/pyasm/web/cherrypy30_startup.py

```diff
@@ -39,6 +39,26 @@ def index(my):
 
 
 
+class TacticIndex:
+    '''Dummy Index file'''
+    def index(my):
+        # check if this project exists
+        response = cherrypy.response
+        request = cherrypy.request
+        path = request.path_info
+
+        from pyasm.security import Site
+        default_project = Site.get().get_default_project()
+        if not default_project:
+            default_project = "admin"
```

Comment:
Not really sure when this TacticIndex is actually called.  It is mostly harmless, but could warrant further investigation.